pkg/execution/state/state.go

```diff
@@ -104,6 +105,50 @@ type CustomConcurrency struct {
 	Limit int `json:"l"`
 }
 
+func (c CustomConcurrency) Validate() error {
```

Comment:
want: test coverage

Reply:
(We'll do this later.)

Reply:
Moved to https://linear.app/inngest/issue/INN-3249/improve-test-coverage-for-customconcurrencyvalidate